marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx

```diff
@@ -254,7 +254,7 @@ const SelectedFacets: React.FC<Props> = (props) => {
           </HCTooltip>
         );
       })}
-      {props.greyFacets.length > 0 &&
+      {Object.keys(greyedOptions.selectedFacets).length > 0 &&
```

Comment:
In this case @littlecherro, could be possible to add an extra control before the ".lenght" to check an undefined?